packages/components/tooltip/src/tooltip.css

```diff
@@ -0,0 +1,61 @@
+:host {
+	position: relative;
+	cursor: pointer;
+	&:before {
+		position: absolute;
+		display: inline-block;
+		align-items: center;
+		background: var(--ts-color-black);
+		border-radius: var(--ts-radius);
+		color: var(--ts-color-white);
+		padding: 0 var(--ts-unit-half);
+		height: var(--ts-unit-double);
```

Comment:
It is a single line tooltip, right?

Reply:
Yes, we can't break it to next line right now.